tests/javaRosa.js

```diff
@@ -463,6 +463,29 @@ require([
             assert.equal(hinLabel[0].selectionStart, 0);
             assert.equal(hinLabel[0].selectionEnd, 15);
         });
+
+        it("should allow user to view longs but not add them", function() {
+            util.loadXML("");
+
+            // able to programatically add long
+            util.addQuestion("Long", "long");
+            util.addQuestion("Int", "int");
+            util.assertJSTreeState(
+                "long",
+                "int"
+            );
+
+            // can't change another question to a long
+            var $changer = $(".fd-question-changer");
+            $changer.children("a").click();
+            assert.equal($changer.find("[data-qtype='Text']").length, 1);
+            assert.equal($changer.find("[data-qtype='Long']").length, 0);
+
+            // can't add new long with toolbar
+            var $toolbar = $(".fd-container-question-type-group");
+            assert.equal($toolbar.find("[data-qtype='Text']:not(.btn)").length, 1); // dropdown item, not button
+            assert.equal($toolbar.find("[data-qtype='Long']").length, 0);
+        });
```

Comment:
Heh, didn't realize this test was added to `tests/javaRosa.js` where it really doesn't belong. The javaRosa module is concerned with itext. It should be moved to `tests/questionTypes.js`.